src/test/e2e/commands_test.go

```diff
@@ -94,6 +94,13 @@ func deploy(t *testing.T, tarballPath string) (stdout string, stderr string) {
 	return stdout, stderr
 }
 
+func deployCustom(t *testing.T, input string) (stdout string, stderr string) {
```

Comment:
Nice! I like this pattern, it can extended beyond just deploying. What do you think of renaming it to something like `runCmd`?